### PR TITLE
Add an example

### PR DESCRIPTION
**Pull request by xiaomi7732**
* .NET 5
* Using Application Insights SDK with connection string
* Docker: Simplified dockerfile to build the binary locally
* K8s: Example of using NodePort for service to debug locally

### CODE REVIEW

File: examples/BuildOutOfContainer/BuildOutOfContainer.csproj [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    <TargetFramework>net5.0</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.ApplicationInsights.AspNetCore" Version="2.17.0" />
+    <PackageReference Include="Swashbuckle.AspNetCore" Version="5.6.3" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\..\src\ApplicationInsights.Kubernetes\ApplicationInsights.Kubernetes.csproj" />
```
Comment (karolz-ms) [inline examples/BuildOutOfContainer/BuildOutOfContainer.csproj:13]: Consider forward slash if you want the project to be usable on Mac/Linux

File: examples/BuildOutOfContainer/Startup.cs [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+        }
+
+        public IConfiguration Configuration { get; }
+
+        // This method gets called by the runtime. Use this method to add services to the container.
+        public void ConfigureServices(IServiceCollection services)
+        {
+            // Application Insights connection string is set in appsettings.json
+            services.AddApplicationInsightsTelemetry();
+            // Skip the following lines when it is in Production environment
```
Comment (karolz-ms) [inline examples/BuildOutOfContainer/Startup.cs:26]: Is there an equivalent of `env.IsDevelopment()` that you could use here?

File: examples/BuildOutOfContainer/Readme.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+```shell
+kubectl delete -f .\K8s.yaml
+```
+
+Got:
+> deployment.apps "ai-k8s-webapi" deleted  
+service "ai-k8s-oc-webapi-svc" deleted  
+
+
```
Comment (karolz-ms) [inline examples/BuildOutOfContainer/Readme.md:65]: What about app settings/App Insights configuration, do you want to cover that too?

**Review by karolz-ms — APPROVED** [review]
LGTM, just a few questions/suggestions!

File: examples/BuildOutOfContainer/BuildOutOfContainer.csproj (hunk 1 above)
Comment (xiaomi7732) [reply examples/BuildOutOfContainer/BuildOutOfContainer.csproj:13]: Having a little bit faith in `csc` :-) It understands the path for all OSes :-) Verified it on Ubuntu. Probably won't fix this. Either unfair to Windows Users or unfair to Unix users :-)

File: examples/BuildOutOfContainer/Startup.cs (hunk 2 above)
Comment (xiaomi7732) [reply examples/BuildOutOfContainer/Startup.cs:26]: The way I can think of is injecting `IWebHostEnvironment`. Will update.

File: examples/BuildOutOfContainer/Readme.md (hunk 3 above)
Comment (xiaomi7732) [reply examples/BuildOutOfContainer/Readme.md:73]: Sure. Why not. :-) Updated the doc. Thanks for the feedback.

File: examples/BuildOutOfContainer/Startup.cs (hunk 2 above)
Comment (xiaomi7732) [reply examples/BuildOutOfContainer/Startup.cs]: Fixed.

**Pull request merged**